eessi_bot_job_manager.py

```diff
@@ -54,28 +53,18 @@ def get_current_jobs(self):
         username = os.getlogin()
 
         squeue_cmd = "%s --long --user=%s" % (self.poll_command, username)
-        log(
-            "get_current_jobs(): run squeue command: %s" % squeue_cmd,
-            self.logfile,
-        )
-
-        squeue = subprocess.run(
+        squeue_output, squeue_err, squeue_exitcode = run_cmd(
             squeue_cmd,
```

Comment:
@Hafsa-Naeem To make sure that logging is done in the right log file, we need to pass down `self.logfile` here:

```python
        squeue_output, squeue_err, squeue_exitcode = run_cmd(
            squeue_cmd,
            "get_current_jobs(): squeue command",
            log_file=self.logfile,
        )
```

Same for the other `run_cmd` calls in `eessi_bot_job_manager.py`